include/deal.II/grid/grid_out.h

```diff
@@ -1080,6 +1080,25 @@ class GridOut
                   std::ostream                      &out) const;
 
   /**
+   * Write triangulation in VTU format for each processor, then combine these
+   * files into a .pvtu file. The input <tt>view_levels</tt> is set to false by
+   * default, allowing only the active level cells to be written into the .vtu
+   * file, but can be set as true to view each level of a multilevel method.
+   * The input <tt>include_artificial</tt> is also set to false by default,
+   * allowing the processor to only write in owned and ghost cells into its .vtu
+   * file, but can be set as true to view the artificial cells for each processor.
+   * Each .vtu and .pvtu file will have the attributes subdomain, lvlsubdomain, level,
+   * and proc_writing. The level value can be used to seperate the image into the
+   * view of the gird on each level of a multilevel method and the proc_writing value
+   * can be used to seperate the image into each processors owned and ghost cells.
+   */
+  template <int dim, int spacedim>
+  void write_debug_vtu_per_processor (const Triangulation<dim,spacedim> &tria,
+                                      std::string                       filename,
```

Comment:
This is not the filename but the base of the filename without the extension, right? Can you mention in the documentation what the names of the files are going to be? Maybe change the name of the argument too?